### PR TITLE
214 and time tracker model updates

### PR DESCRIPTION
**Pull request by rachelwhaley**
Adding videoconference link field for #214 and adding time tracker model re: #192 (cc @patrickgus)

### CODE REVIEW

File: models/timeTracker.model.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            userId: this.user.userId
+        },
+        selectedAnswer: this.selectedAnswer,
+        createdDate: this.createdDate
+    };
+};
+
+const TimeTracker = mongoose.model('TimeTracker', timeTrackerSchema);
+
+module.exports = { timeTracked };
```
Comment (alex-anakin) [inline models/timeTracker.model.js]: please add an empty line at the end

File: models/timeTracker.model.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+
+const timeTrackerSchema = mongoose.Schema({
+    hours: { type: Number },                // limit to whole numbers or .5 ?
```
Comment (alex-anakin) [inline models/timeTracker.model.js]: it's better to calculate hours in the code using the start time and end time. Let's remove this field.

File: models/timeTracker.model.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User'
+    },
+    project: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Project'
+    },
+    category: { type: String },             // picklist with 4 options: Development, Design/UX, Product/Project Management, Other
+    notes: { type: String },
+    createdDate: { type: Date, default: Date.now }
```
Comment (alex-anakin) [inline models/timeTracker.model.js]: please add 2 Date fields: startDate, endDate (without default value)

**Review by alex-anakin — CHANGES_REQUESTED** [review]
Please resolve conflict (apply changes from server to README) and resolve my comments.

File: models/timeTracker.model.js (hunk 2 above)
Comment (nickbeaird) [reply models/timeTracker.model.js]: ++

File: models/timeTracker.model.js (hunk 3 above)
Comment (nickbeaird) [reply models/timeTracker.model.js]: ++

**Comment by patrickgus** [comment]
@alex-anakin requested changes were made

**Comment by nickbeaird** [comment]
It looks like this PR pulled in a ton of prior commits. Is there a way to clean it up, so that it has just your changes? @patrickgus

**Comment by patrickgus** [comment]
@nickbeaird Yeah I'm not sure why it pulled so many commits but you can filter the 'files changed' by the individual commits. So mine are the last 3: 4d1e17a, 980a1c5, and 1cee393.

**Comment by nickbeaird** [comment]
Hey @patrickgus, I'd prefer it if you can please resubmit the PR/branch if that's the case. I can either review this PR, or the new one then. Also, if you are having any issues getting a branch setup without the commits, then let me know and I can try and help out.

**Comment by patrickgus** [comment]
@nickbeaird Should I just delete this PR entirely and create a new one?

**Comment by nickbeaird** [comment]
Yeah, I would just make the branch and when you feel good with the new PR, then close this one.

**Comment by patrickgus** [comment]
@nickbeaird Just created a new PR #302 that shows only the recent changes. I am closing this one.

**Pull request closed without merging**